src/cljs/main/broadfcui/common.cljs

```diff
@@ -210,3 +210,26 @@
                 :alert)
     [:span {:style {:fontWeight "bold" :fontSize "98%"}}
       "FireCloud is not intended to host personally identifiable information. Do not use any patient identifier, including name, social security number, or medical record number."]])
+
+(react/defc FoundationComponent
+  {:component-did-mount
+   (fn [{:keys [this]}]
+     (.foundation (js/$ (react/find-dom-node this))))
+   :render
+   (fn [{:keys [props]}]
+     (:contents props))})
+
+(react/defc FoundationTooltip
+  {:render
+   (fn [{:keys [props]}]
+     (let [rand-id (gensym "tooltip-")
+           position (:position props)]
+       [:div {}
+        [:button {:className "has-tip" :data-toggle rand-id}
+         (:text props)]
+        [FoundationComponent
+         {:contents [:div {:className (str "tooltip " position) :id rand-id :data-dropdown ""
+                           :data-v-offset (if (or (not position) (= position "top")) 12 100)
+                           :data-h-offset (if (or (= position "left") (= position "right")) 12 100)
+                           :data-hover true :data-hover-pane true}
+                     (:tooltip props)]}]]))})
```

Comment:
(Comment is meant to be attached to the whole component, not just this last line)

Recently I've been unpacking `props` fully.  I find it helps document what the component's properties are.  So in that `let`, instead of just pulling out `position`, do `{:keys [position text tooltip]} props` and then use those.

It would also be good to have a default `:position` prop, both for ease of use and to document what its valid values look like.